src/Core/Prefab/PanoramaView.js

```diff
@@ -157,6 +157,8 @@ export function createPanoramaLayer(id, coordinates, ratio, options = {}) {
     tileLayer.type = 'geometry';
     tileLayer.protocol = 'tile';
     tileLayer.visible = true;
+    tileLayer.segments = 8;
+    tileLayer.quality = 0.5;
```

Comment:
Could you comment how you choose quality? sort of Screen-Space Error?

Reply:
The tiles are refined until the `screen_space_size * quality` is >= texture pixel size.